.gitignore

```diff
@@ -1,2 +1,3 @@
 talon_one-*.gem
 secrets.yml
+Gemfile.lock
```

Comment:
The [internet disagrees](https://www.google.de/search?q=checking+in+gemfile.lock)